Common/Model.cs

```diff
@@ -88,6 +74,13 @@ private void LoadModel(string path)
             // is now contained within our list of processed meshes
             importer.Dispose();
         }
+
+        // Draws the model, and thus all its meshes
+        public void Draw()
+        {
+            for (int i = 0; i < meshes.Count; i++)
+                meshes[i].Draw();
+        }
```

Comment:
I would actually like to see this logic moved into the chapter code itself. This type of abstraction using objects with a `Draw` method isn't an abstraction that works in the long term for graphics. It becomes really inefficient, so I think that we should avoid making the suggestion that this is a good way to structure rendering code.

Reply:
This means both the mesh and the model drawing code.

And I know the other abstractions like texture and shaders do this type of things too with their `Use` methods, but I'm not happy with those either.